recipe/meta.yaml

```
@@ -1,17 +1,22 @@
-{% set unix_version = "5.34.0" %}
+{% set unix_version = "5.38.2" %}
+{% set win_version = unix_version + ".2" %}
+{% set concat_version = win_version.split('.') | join('') %}
 
 package:
   name: perl
-  version: {{ unix_version }}
+  version: {{ unix_version }}  # [not win]
+  version: {{ win_version }}   # [win]
 
 source:
-  url: https://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.gz                                                 # [unix]
-  sha256: 551efc818b968b05216024fb0b727ef2ad4c100f8cb6b43fab615fa78ae5be9a                                        # [unix]
+  - url: http://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.xz                                                 # [unix]
+    sha256: d91115e90b896520e83d4de6b52f8254ef2b70a8d545ffab33200ea9f1cf29e8                                        # [unix]
+  - url: https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/download/SP_{{ concat_version }}_64bit/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win]
+    sha256: ea451686065d6338d7e4d4a04c9af49f17951d15aa4c2e19ab8cb56fa2373440                                        # [win]
+    # sha hashes published at: http://strawberryperl.com/releases.html
 
 build:
-  number: 2
-  # there is no Windows strawberry release of this version
-  skip: True  # [win]
+  number: 0
```

```suggestion
  number: 0
  string: {{ PKG_BUILDNUM }}_h{{ PKG_HASH }}_strawberry  # [win]
  string: {{ PKG_BUILDNUM }}_h{{ PKG_HASH }}_perl5       # [not win]
```
This was the pattern used before the windows build was removed.